support/cas-server-support-oauth/src/main/java/org/apereo/cas/config/CasOAuth20Configuration.java

```diff
@@ -1158,9 +1158,10 @@ public ExpirationPolicyBuilder refreshTokenExpirationPolicy(final CasConfigurati
         public OAuth20RefreshTokenFactory defaultRefreshTokenFactory(
             @Qualifier("refreshTokenIdGenerator") final UniqueTicketIdGenerator refreshTokenIdGenerator,
             @Qualifier("refreshTokenExpirationPolicy") final ExpirationPolicyBuilder refreshTokenExpirationPolicy,
-            @Qualifier(ServicesManager.BEAN_NAME) final ServicesManager servicesManager) {
+            @Qualifier(ServicesManager.BEAN_NAME) final ServicesManager servicesManager,
+            final CasConfigurationProperties casProperties) {
             return new OAuth20DefaultRefreshTokenFactory(refreshTokenIdGenerator,
-                refreshTokenExpirationPolicy, servicesManager);
+                refreshTokenExpirationPolicy, servicesManager, casProperties.getLogout().isRemoveDescendantTickets());
```

Comment:
I think we did remove this property and replaced it with a ticket-tracking mechanism/setting right? The property should be adjusted I think.

Reply:
Yes, it should be adjusted but I submitted this PR before the https://github.com/apereo/cas/pull/5858 was merged. Will do